index.js

```diff
@@ -40,6 +64,14 @@ function sanitizeComponent(cmpt) {
   }
 }
 
+/**
+ * render
+ *
+ * formats an amphora-resolved component for Apple News
```

Comment:
Why does the comment say `amphora-resolved` component but the function looks like it's only concerned with an `article` component based on naming of variables?

Reply:
because it will eventually handle videos and galleries..?